src/handleImportMap.test.ts

```diff
@@ -23,11 +23,88 @@ describe(`handleImportMap`, () => {
   });
 
   it(`returns a 404 Not Found if the map isn't in KV`, async () => {
-    const request = new Request("https://cdn.example.com");
+    (global.MAIN_KV as MockCloudflareKV).mockKv({});
+
+    const request = new Request("https://cdn.example.com/systemjs.importmap");
     const response: Response = await handleImportMap(request, {
       importMapName: "system",
       orgKey: "juc",
     });
     expect(response.status).toBe(404);
   });
+
+  it(`returns a 500 if importMap.imports`, async () => {
```

Comment:
It's not clear to me what the assert description means. Is it missing some language? Are we trying to import a "string" value?

Reply:
👍🏿  good catch i updated the test name